lib/components/viewers/realtime-status-label.tsx

```diff
@@ -96,7 +96,7 @@ const RealtimeStatusLabel = ({
     // and display the updated time underneath.
     renderedTime = isEarlyOrLate ? (
       <TimeBlock>
-        <TimeStruck>
+        <TimeStruck aria-hidden>
```

Comment:
IMO, we still need to say what the original time was.

Reply:
This actually feeds into the work we're doing on https://github.com/opentripplanner/otp-ui/pull/549. If we go with the below description, how would we incorporate the old time? 

`6:44 pm, from <location>, ride 4 stops to <next location>`

Something like?

`6:44 pm (4 minutes late, previously 6:40PM), from <location>, ride 4 stops to <next location>`

Reply:
Indeed. Maybe "originally" instead of "previously".